pkg/token/service.go

```diff
@@ -41,9 +41,11 @@ type tokenService struct {
 	shutdownChan chan struct{}
 	shutdownWg   sync.WaitGroup
 
-	group            singleflight.Group
-	accessTokenCache TokenCache
-	roleTokenCache   TokenCache
+	group               singleflight.Group
+	accessTokenCache    TokenCache
+	roleTokenCache      TokenCache
+	atTargetDomainRoles []CacheKey
+	rtTargetDomainRoles []CacheKey
```

Comment:
Write comments what these are, and how they are different from `accessTokenCache` & `roleTokenCache`
I believe these are for the file inputs, but I dont see any comments about the differences unless I read codes.

Reply:
prolly better if variable has something related to writing as files, so that we know these variables for writing files.

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/131/commits/1e2206eb193c0553c182a0d487bf41e1f03ff7e5